### PR TITLE
Ft-vet_clinick

### PR DESCRIPTION
**Pull request by zbahati**
- Add queries filter into data.sql
-Add base queries like insert into queries.sql
- Add base skeleton into schema.sql

#  Finishes ..

## screenshot  of the queries 
neutered
Screenshot (2
![data
37)](https://github.com/zbahati/vet_clinick/assets/103934171/5c201dfe-d9d5-4028-876f-61837aeaed47)
greater than
search_like](https://github.com/zbahati/vet_clinick/assets/103934171/0267368f-6bf3-4a58-85d0-d6679d5362db)
ahati/vet_clinick/assets/103934171/7e5c9d3d-8468-4917-97a0-d2caa00f290c)
Screenshot (240)
Screenshot (243)
Screenshot (244)
Screenshot (246)

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
-);
+CREATE TABLE animals(
+	id INT,
```
Comment (muneebulrehman) [inline schema.sql]: - Optional
I would recommend making the id column not null and autoincremented and also a primary key.

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+-- 2. List the name of all animals born between 2016 and 2019:
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+-- 3. List the name of all animals that are neutered and have less than 3 escape attempts:
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
+
+-- 4. List the date of birth of all animals named either "Agumon" or "Pikachu":
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
```
Comment (muneebulrehman) [inline queries.sql]: This query is not for listing the date_of_birth of all animals named either "Agumon" or "Pikachu". Kindly write a valid query and include screenshot of the results in your PR.

File: queries.sql [hunk 3]
```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- 1. Find all animals whose name ends in "mon":
+SELECT * FROM animals WHERE name LIKE '%mon'
+
+-- 2. List the name of all animals born between 2016 and 2019:
+
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
```
Comment (muneebulrehman) [inline queries.sql:8]: Kindly include the screenshot for the results of this query.

File: queries.sql [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+
+-- 3. List the name of all animals that are neutered and have less than 3 escape attempts:
+SELECT name FROM animals WHERE neutered = true AND escape_attempts < 3;
```
Comment (muneebulrehman) [inline queries.sql:11]: You have not included the screenshot for this query. Kindly do so.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @zbahati , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR

## Required Changes ♻️
_Check the comments under the review._

- [ ] You have not included the queries for these two quests. ⬇️
image

Kindly write the queries for the same and remember to include the screenshots of the results in the PR description.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 3 above)
Comment (danmainah) [reply queries.sql:8]: - [ ] @zbahati Great work implementing some of the changes requestedd, however you forgot to add a screenshot for this query kindly added it to your pull request description.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @zbahati , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️  Added all the queries for this milestone successfully.

 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @zbahati 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
Added missing screenshot 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**